core/build.gradle

```diff
@@ -67,4 +69,9 @@ dependencies {
     testImplementation "org.assertj:assertj-core:$assertj_version"
     testAnnotationProcessor "com.google.dagger:dagger-compiler:$dagger_version"
     testAnnotationProcessor "com.google.dagger:dagger-android-processor:$dagger_version"
+    compile "androidx.core:core-ktx:+"
```

Comment:
We don't version with `+ `.  Put the latest stable version of this library. Also, `compile` is deprecated. You should use `implementation`.